index.html

```diff
@@ -0,0 +1,80 @@
+                <a><img alt="Logo"  src="Assets/Images/my_logo.png"></a>
+            </nav>
+            <div class="menu_btn">
+                <img alt="Menu_btn_element" class="menu_btn_elm" src="Assets/Images/menu.png">
+                <img alt="Menu_btn_element" class="menu_btn_elm" src="Assets/Images/menu.png">
+                <img alt="Menu_btn_element" class="menu_btn_elm" src="Assets/Images/menu.png">
+            </div>
+        </header>
+        <div class="content_body">
+            <div class="greetings">
+                <h1>I'm Mario Glad to see you!</h1>
+            </div>
+            <div class="about">
+                <h3>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3>
+            <div class="lets_connect">
+                <a>LET'S CONNECT</a>
+            </div>
+            </div>
+        </div>
+        <section class="footer">
+            <ul class="footer_ul">
+                <li>
+                    <a>
+                        <img alt="twiter" class="element" src="Assets/Images/twiter.png">
+                    </a>
+                </li>
+                <li>
+                    <a>
+                        <div class="element_else">
+                            <div class="left_side">
+                                <img alt="in1" class="img1" src="Assets/Images/in1.png">
+                                <img alt="in2" class="img2" src="Assets/Images/in3.png">
+                            </div>
+                            <div class="right_side">
+                                <img alt="in3" src="Assets/Images/in2.png" >
+                            </div>
+                        </div>
+                    </a>
+                </li>
+                <li>
+                    <a>
+                        <img alt="m" class="element" width="7" src="Assets/Images/m.png">
+                    </a> 
+                </li>
+                <li>
+                    <a>
+                        <img alt="github" class="element" width="7" src="Assets/Images/github.png">
+                    </a>
+                </li>
+                <li>
+                    <a>
+                        <img alt="fingers" class="element" width="7" src="Assets/Images/fingers.png">
+                    </a>
+                </li>
+            </ul>         
+        </section>
+    </div>
+    <div>
+        <img alt="Element1" class="elm_1" src="Assets/Images/elm1.png">
+        <img alt="Element2"  class="elm_2" src="Assets/Images/elm2.png">
+        <img alt="Element3"  class="elm_3" src="Assets/Images/elm3.png">
+        <img alt="Element4"  class="elm_4" src="Assets/Images/elm4.png">
+        <img alt="Element5"  class="elm_5" src="Assets/Images/elm5.png">
+        <img alt="Element6"  class="elm_6" src="Assets/Images/elm6.png">
```

Comment:
I suggest you use a background image for decorations as mentioned in the project requirement (decorations are always background images, not img tags)